game/src/Network/ServerGameMode.cpp

```diff
@@ -445,6 +445,33 @@ void ServerGameMode::onCollisionExit(Entity e1, Entity e2) {
   }
 }
 
+int ServerGameMode::getNearestPlayer(const int& ent)
+{
+    int returnIndex = 0;
+    float nearestLenght = glm::length(this->getComponent<Transform>(ent).position - this->getComponent<Transform>(players[0]).position);
```

Comment:
Can you compare the squared lengths instead of the actual lengths, to avoid the square roots?